packages/wrangler/src/dev/dev.tsx

```diff
@@ -244,7 +255,55 @@ type DevSessionProps = DevProps & {
 };
 
 function DevSession(props: DevSessionProps) {
```

Comment:
This is (_essentially_) the entry-point to `wrangler dev` – here we create a `DevEnv` and map the props into `StartDevWorkerOptions` (at least what is relevant for now – for ProxyController) and fake events by calling `ProxyController` event handler methods (`devEnv.proxy.on*`)